<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentSuccess</class>
 <widget class="QMainWindow" name="StudentSuccess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(159, 166, 170);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="signUpBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>311</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);
border-radius: 10px;</string>
    </property>
    <property name="title">
     <string>Create Your Account</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="name">
       <property name="minimumSize">
        <size>
         <width>293</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
border-radius: 5px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="email">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
border-radius: 5px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="major">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
border-radius: 5px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="id">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
border-radius: 5px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="semester">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
border-radius: 5px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="gpa">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="signUp">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#signUp{
background-color: rgb(85, 170, 0);
font: 700 14pt &quot;Segoe UI&quot;;
border-radius: 5px;
border-width:0px;
}

#signUp:hover{
	background-color: rgb(0, 170, 127);
}</string>
       </property>
       <property name="text">
        <string>Signup</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="loginBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>261</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);
border-radius: 10px;</string>
    </property>
    <property name="title">
     <string>Please login:</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="studentEmail">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
border-radius: 5px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="StudentId">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="login">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#login{
background-color: rgb(85, 170, 0);
font: 700 14pt &quot;Segoe UI&quot;;
border-radius: 5px;
border-width:0px;
}

#login:hover{
	background-color: rgb(0, 170, 127);
}</string>
       </property>
       <property name="text">
        <string>login</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="moto">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>651</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 24pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="plainText">
     <string>Student Success Empowers 
Your Student Journey...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>email</tabstop>
  <tabstop>major</tabstop>
  <tabstop>id</tabstop>
  <tabstop>semester</tabstop>
  <tabstop>gpa</tabstop>
  <tabstop>moto</tabstop>
  <tabstop>login</tabstop>
  <tabstop>signUp</tabstop>
  <tabstop>studentEmail</tabstop>
  <tabstop>StudentId</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
